securing_apps/topics/oidc/java/servlet-filter-adapter.adoc

```diff
@@ -52,6 +52,16 @@ Note that you should configure your client in the {project_name} Admin Console w
 The Admin URL will make callbacks to the Admin URL to do things like backchannel logout.
 So, the Admin URL in this example should be `http[s]://hostname/{context-root}/keycloak`.
 
+If you need to customize the session ID mapper you can configure the fully qualified name of the class in the Filter init-param keycloak.config.idMapper. Session ID mapper is a mapper which is used to map user IDs and session IDs. By default org.keycloak.adapters.spi.InMemorySessionIdMapper is configured.
```

Comment:
comma after an introductory clause -- after mapper,
"is a mapper that..." - use "that' not "which" when the rest of the sentence qualifies the noun, meaning it is the specific mapper "that" is used to map user IDs and session IDs.